apps/ledger-live-mobile/src/index.tsx

```diff
@@ -231,9 +230,7 @@ const StylesProvider = ({ children }: { children: React.ReactNode }) => {
 
   return (
     <StyleProvider selectedPalette={resolvedTheme}>
-      <GeneralTermsContextProvider>
```

Comment:
this provider was killed because all the data needed live in the redux store (cf. changes in `logic/terms.tsx`)